### DISCUSSION TITLE
Plans for topology optimization?

### DISCUSSION THREAD

**Discussion by senpaibernd** [OpenRadioss/OpenRadioss#616] [Q&A] [answered]
Does anyone know if there are plans to make the topology optimizer (OptiStruct) open source as well?

Yesterday i tried to do the example RD-E: 5102: 
[https://2021.help.altair.com/2021/hwsolvers/rad/topics/solvers/rad/optimization_radioss_solids_example_r.htm#optimization_radioss_solids_example_r_reference_lvk_zt4_ldb](url)

...but noticed that OptiStruct is additionally necessary.  🙄

**To include an topology optimizer would make this solution very complete!**

**Comment by PaulAltair** [comment, marked as answer]
No, sorry, there are no plans to make OptiStruct open source.

**Reply by senpaibernd** [reply]
OK. Thanks for this information.
